include/libtorrent/aux_/time.hpp

```diff
@@ -43,6 +43,9 @@ namespace libtorrent { namespace aux {
 	// resolution of this timer is about 100 ms.
 	TORRENT_EXTRA_EXPORT time_point time_now();
 	TORRENT_EXTRA_EXPORT time_point32 time_now32();
+
+	TORRENT_EXTRA_EXPORT std::time_t to_time_t(const time_point32 &tp);
```

Comment:
I don't think there's a good reason to pass this parameter by reference. it's only a 32 bit integer.